### DISCUSSION TITLE
1 forum to rule 'em all

### DISCUSSION THREAD

**Discussion by jf---** [compas-dev/compas#1317] [Ideas]
compas also has the [discourse](https://forum.compas-framework.org/) forum
many questions deal with interaction across compas modules.
hence I think it would be best to have 1 forum rather than many diluted fora

discourse is just absolute bliss IMO

here's [how `discussion` could be disabled in a repo](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/enabling-or-disabling-github-discussions-for-a-repository)

**Comment by yck011522** [comment]
My 2 cents also support using one forum to consolidate.

**Reply by gonzalocasas** [reply]
Is there an easy way to convert a github issue into a discourse discussion?
